test/libsolidity/syntaxTests/structs/calldata_assign.sol

```diff
@@ -0,0 +1,8 @@
+pragma experimental ABIEncoderV2;
+contract Test {
+    struct S { int a; }
+    function f(S calldata s) external { s.a = 4; }
```

Comment:
`` test/libsolidity/syntaxTests/structs/memory_to_calldata.sol``
Should I try to produce a better error message than ``Expression has to be an lvalue``? Not marking as lvalue was the easiest way to trigger an error on writes, but it might in fact not be the clearest error message...

Reply:
It's fine for now, I would say.